app/routes/events/EventDetailRoute.js

```diff
@@ -125,10 +125,9 @@ const mapDispatchToProps = {
   updateUser
 };
 
-const loadData = ({ params: { eventId }, currentUser, loggedIn }, dispatch) => {
-  const userId = currentUser.id;
+const loadData = ({ params: { eventId }, loggedIn }, dispatch) => {
   return dispatch(fetchEvent(eventId)).then(
```

Comment:
Make these two concurrent while fixing stuff here (and remember to handle errors). No need to wait for fetchEvent